### DISCUSSION TITLE
How can I know that my  organization has access to Codespace VM

### DISCUSSION THREAD

**Discussion by SBU2023** [community/community#64908] [GitHub Education]
### Hi everyone,

I am teaching two courses at Stony Brook University. I am planning to use Github classroom for the assignment management. My plan is to enable codespace for each organization so that when the student access the assignment, they can use the code space as well. How can I ensure that is available? My organizations have been approved for teacher benefits. The message also says that the benefits will appear soon. How long it usually take to have the "teacher benefits"?

**Comment by nirav-gajera** [comment]
To ensure that codespaces are available for your students, you need to enable them for each organization. You can do this by following these steps:

1. Go to the GitHub Classroom settings page for your organization.
2. Click on the "Codespaces" tab.
3. Check the box next to "Enable codespaces for this organization".
4. Click on the "Save" button.

Once you have enabled codespaces for your organization, students will be able to use them

The benefits usually appear within 1-2 business days after your application is approved. However, it may take longer if there is a high volume of applications.

**Reply by SBU2023** [reply]
Thanks. I got the approval. However, I still can not see the Codespaces in action with the assignment. I can not enable it when I go to setting-->Codespacces--> ( I can see the enabling option but I cannot turn it on). Quick reply is requested as the semester is almost started.

**Reply by SBU2023** [reply]
<img width="386" alt="Screenshot 2023-08-28 155355" src="https://github.com/community/community/assets/140561879/46008a11-a214-4c2d-a171-57a805abe3a2">
This is what I am getting when testing as a student.

**Reply by SBU2023** [reply]
Can someone guide me on this? I have the teacher benefit approval on both my organizations. However, I am unable to link Codespaces with them for my student assignments.

**Reply by SBU2023** [reply]
Can someone guide on this? How the students can use Codesspaces with this? Classes are almost started and I am not clear on this.

**Comment by SBU2023** [comment]
Any feedback is helpful. I need to make a decision to move forward with my class.

**Comment by SBU2023** [comment]
<img width="918" alt="Screenshot 2023-09-01 165029" src="https://github.com/community/community/assets/140561879/542bc6f9-12c3-4d72-ab25-893ae98ee9ec">
Why I see this on my organization?
